### ISSUE TITLE
CLI options to watch a directory

### ISSUE THREAD

**Issue by ysimonson** [sass/node-sass#1212]
Hello, I've been trying to figure out what the proper command line arguments are to watch a directory. I tried several combos - all of which seem to do keep the CLI running, but not doing anything (files don't compile, nothing in stdout/stderr.)

Based off #700 , I tried this: `node-sass -w -r --output static/stylesheets static/sass`

Based off #1075, I tried this: `node-sass -w web/static/sass -o web/static/stylesheets`

**Comment by xzyfer** [comment]
Are you watching `.sass` or `.scss` files. There is a known issue with the watcher and `.sass` files currently.

**Comment by xzyfer** [comment]
Possibly related https://github.com/sass/node-sass/issues/1179

**Comment by ysimonson** [comment]
It's `.scss` files

**Comment by xzyfer** [comment]
```
node-sass [options] <input> [output]
```
So the answer to your questions is
```
node-sass -w -r web/static/sass web/static/stylesheets
```
If this doesn't work please create a small, fully working, github repo that shows your problem.

**Comment by ysimonson** [comment]
@xzyfer that gives me the error: `An output directory must be specified when compiling a directory` (it actually prints it out twice), even though web/static/stylesheets exists.

**Comment by xzyfer** [comment]
This should be fixed by #1322.

**Comment by xzyfer** [comment]
This is fixed on master and will be in the next release.
